lib/fixup.rb

```diff
@@ -4,30 +4,107 @@
 require 'json'
 require_relative 'color'
 require_relative 'package'
-require_relative 'convenience_functions'
+
+def require_gem(gem_name_and_require = nil, require_override = nil)
+  # Allow only loading gems when needed.
+  return if gem_name_and_require.nil?
+
+  gem_name = gem_name_and_require.split('/')[0]
+  begin
+    gem gem_name
+  rescue LoadError
+    puts " -> install #{gem_name} gem".orange
+    Gem.install(gem_name)
+    gem gem_name
+  end
+  requires = if require_override.nil?
+               gem_name_and_require.split('/')[1].nil? ? gem_name_and_require.split('/')[0] : gem_name_and_require
+             else
+               require_override
+             end
+  require requires
+end
+require_gem('highline')
 
 # All needed constants & variables should be defined here in case they
 # have not yet been loaded or fixup is being run standalone.
 
-CREW_VERBOSE = ARGV.intersect?(%w[-v --verbose]) unless defined?(CREW_VERBOSE)
+CREW_VERBOSE ||= ARGV.intersect?(%w[-v --verbose]) unless defined?(CREW_VERBOSE)
```

Comment:
You don't need `unless defined?(CREW_VERBOSE)` if you're using `||=`, do you?

Reply:
You would think not, but for some reason for the way these variables are being set ruby complains otherwise. 🤷🏿

Reply:
I thought it was annoying too...

Every time I have both is because the `||=` wasn't sufficient.